src/chapters/jdbc-auth.xml

```diff
@@ -70,12 +70,34 @@
                             <filename>GUACAMOLE_HOME/extensions</filename>, while the MySQL JDBC
                         driver must be placed within <filename>GUACAMOLE_HOME/lib</filename>.</para>
                     <para><emphasis>The MySQL JDBC driver is not included with the
-                            extension.</emphasis> You must obtain the JDBC driver
-                            <filename>.jar</filename> yourself from <link
-                            xlink:href="http://dev.mysql.com/downloads/connector/j/">MySQL's
-                            website</link>. The driver is known as "Connector/J", and the required
+                            extension.</emphasis> You must obtain a supported JDBC driver
+                            <filename>.jar</filename> yourself.  The MySQL driver can be downloaded
+							from <link xlink:href="http://dev.mysql.com/downloads/connector/j/">MySQL's
```

Comment:
Looks like the same space vs. tab issue noted in #136 has occurred here.

Reply:
Doh.  Should be fixed.